client/src/pages/Dashboard/EditProfilePage.tsx

```diff
@@ -0,0 +1,281 @@
+import Button from '@material-ui/core/Button';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import TextField from '@material-ui/core/TextField';
+import FormControl from '@material-ui/core/FormControl';
+import Box from '@material-ui/core/Box';
+import Grid from '@material-ui/core/Grid';
+import TextareaAutosize from '@material-ui/core/TextareaAutosize';
+import Typography from '@material-ui/core/Typography';
+import { makeStyles } from '@material-ui/core/styles';
+import Container from '@material-ui/core/Container';
+import MuiPhoneNumber from "material-ui-phone-number";
+
+const theme = {
+    spacing: 8,
+};
+ 
+  
+const useStyles = makeStyles((theme) => ({
+paper: {
+    marginTop: theme.spacing(8),
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+},
+
+form: {
+    width: '100%', // Fix IE 11 issue.
+    marginTop: theme.spacing(3),
+},
+submit: {
+    margin: theme.spacing(3, 0, 2),
+    padding: "0.5rem",
+    width: "10%"
+},
+textbox: {
+    width: '100%'
+},
+typography:{
+    fontWeight: "550",
+    fontSize: "18px"
+},
+gridcontainer:{
+    padding: theme.spacing(1)
+},
+heading:{
+    fontWeight: "550"
+}
+}));
+ 
+ const EditProfile = () => {
+
+    const classes = useStyles();
+    const phoneRegExp = /^((\\+[1-9]{1,4}[ \\-]*)|(\\([0-9]{2,3}\\)[ \\-]*)|([0-9]{2,4})[ \\-]*)*?[0-9]{3,4}?[ \\-]*[0-9]{3,4}?$/;
+   return (
+     <Formik
+       initialValues={{ firstName: '', lastName: '', email: '', birthdate: '', address: '', phoneNumber: '', description: '' }}
+       validationSchema={Yup.object({
+         firstName: Yup.string()
+           .max(15, 'Must be 15 characters or less'),
+         lastName: Yup.string()
+           .max(20, 'Must be 20 characters or less'),
+         email: Yup.string().email('Invalid email address').required('Required'),
+         phoneNumber: Yup.string().matches(phoneRegExp, 'Phone number is not valid')
+
+       })}
+       onSubmit={(values, { setSubmitting }) => {
+           console.log(values);
+         setTimeout(() => {
+           alert(JSON.stringify(values, null, 2));
+           setSubmitting(false);
+         }, 400);
+         /* //to post data to given url
+         
+            event.preventDefault();
+            const url = 'https://jsonplaceholder.typicode.com/posts'
+            const requestOptions = {
+                method: 'POST',
+                headers: { 'Content-Type': 'application/json' },
+                body: JSON.stringify({ email, password })
+            };
+            fetch(url, requestOptions)
+                .then(response => console.log('Submitted successfully'))
+                .catch(error => console.log('Form submit error', error))
+         */
+       }}
+     >
+{formik => (
+    <form className={classes.form} noValidate onSubmit={formik.handleSubmit}>
+    <Container component="main" maxWidth="md">
+      <CssBaseline />
+      <div className={classes.paper}>
```

Comment:
You can use a MUI Box component here instead, it has flexbox built into it.

Ref: https://material-ui.com/components/box/